cli/flox/src/commands/envs.rs

```diff
@@ -188,15 +188,15 @@ fn format_description(env: &UninitializedEnvironment) -> Cow<'_, str> {
 }
 
 fn format_path(path: Option<&Path>) -> Cow<'_, str> {
-    path.map(|p| p.to_string_lossy())
+    path.map(|p| p.parent().unwrap_or(p).to_string_lossy())
```

Comment:
note: this feels like kind of a gotcha *sigh*. But like you said we probably shouldn't clean up more now

Reply:
Yeah this parenting sucks
For errors I guess we could have a convenience method for it or wrap it in a missing env path newtype
Or make sure that this error is only generated by dotflox open and then change the semantics.
That starts scratching at the bigger path back and forth I mentioned earlier though.

For printing we can add a method on dot flox or implement display that strips the .flox for convenience